server/utilities/PokerUtils.js

```diff
@@ -1,7 +1,7 @@
 /**
  * The number of seats at a table
  */
-const NUM_OF_TABLE_SEATS = 6;
+const NUM_OF_TABLE_SEATS = 1;
```

Comment:
Why was this changed, this is a constant stating the maximum number of people that can be playing at a table?

Reply:
It was changed to speed up testing and not require 6 players to test. Will change back to default 6.